src/api/nutrient.api.js

```diff
@@ -11,4 +12,14 @@ nutrientApi.get('/', (req, res) => {
   nutrientController.list(req, res);
 });
 
+nutrientApi.post('/create', async (req, res) => {
```

Comment:
Além disso, a funcionalidade de lista de nutricionistas não precisa ter o comportamento de adicionar um novo. Quem vai adicionar a esta lista somente é o nutricionista quando ele for convidar o paciente para pertencer a ele.